pkg/volume/plugins.go

```diff
@@ -706,30 +706,26 @@ func (pm *VolumePluginMgr) FindPluginByName(name string) (VolumePlugin, error) {
 	defer pm.mutex.RUnlock()
 
 	// Once we can get rid of legacy names we can reduce this to a map lookup.
-	matches := []VolumePlugin{}
+	var matche VolumePlugin
```

Comment:
Does `matche` have a meaning that I don't know about? `macthe` -> `match`?

Reply:
it's my typo, revise now, thanks!